src/qt/forms/debugwindow.ui

```diff
@@ -592,36 +592,13 @@
           <number>3</number>
          </property>
          <item>
-          <widget class="QPushButton" name="promptIcon">
-           <property name="enabled">
-            <bool>false</bool>
-           </property>
+          <widget class="QLabel" name="promptIcon">
            <property name="maximumSize">
             <size>
              <width>16</width>
              <height>24</height>
             </size>
            </property>
-           <property name="text">
-            <string/>
-           </property>
-           <property name="icon">
-            <iconset resource="../bitcoin.qrc">
-             <normaloff>:/icons/prompticon</normaloff>
-             <disabledoff>:/icons/prompticon</disabledoff>:/icons/prompticon</iconset>
```

Comment:
I'd keep setting an icon in the UI file to provide it in the Qt Designer tool.

Reply:
addressed in 82d430d